demo/src/components/Cart/Cart.js

```diff
@@ -1,19 +1,27 @@
 import React from 'react';
 import {connect} from 'react-redux';
 import {Container, Col, Row, Image, Form} from 'react-bootstrap';
-import './Cart.css';
-import {setQuantity} from '../../store/StoreHelpers.js';
+import {setQuantity, removeOneFromCart} from '../../store/StoreHelpers.js';
 import PropTypes from 'prop-types';
-import {computePriceOfItemsInCart} from '../../utils.js';
+import {CodeModal} from '../CodeModal/CodeModal';
+import {FiTrash2} from 'react-icons/fi';
+import './Cart.css';
+import '../CodeModal/CodeModal.css';
+import {computePriceOfItemsInCart, getMeasureCodeSnippet} from '../../utils.js';
+import {getRemoveFromCartCodeSnippet, getViewCartCodeSnippet} from '../../lib/gtagSnippets';
+import {sendRemoveFromCartEvent} from '../../lib/gtagEvents';
 
 /**
  * Creates a component describing a shopping Cart.
  * @param {ItemStore} items The global {@link ItemStore} site object.
  * @param {function(string, number)} setQuantity A function to modify
  *      the quantity of an item in the global state.
+ * @param {function(string)} removeOneFromCart A function to reduce the
+ * quantity of an element in the cart by 1, deleting it if the quantity is
+ * empty.
  * @return {!JSX} The component.
  */
-const CartBase = function({items, setQuantity}) {
+const CartBase = function({items, setQuantity, removeOneFromCart}) {
```

Comment:
nit: I'd prefer arrow functions when possible. Is there a reason not to use one here?

Reply:
Not that I know of, I changed it.